packages/nimbus/src/components/text-input/text-input.mdx

```diff
@@ -13,106 +13,180 @@ tags:
+### Resources
 
-## Features
+Deep dive on details and access design library.
 
-- **Accessible:** Built with ARIA attributes for screen reader support
-- **Customizable:** Supports different sizes and visual variants
-- **State Management:** Handles disabled, invalid, and focus states
+[TECH RELATED LINK](#)
+[TECH RELATED LINK](#)
+[Figma library](https://www.figma.com/design/gHbAJGfcrCv7f2bgzUQgHq/NIMBUS-Guidelines?node-id=1696-48973&m=dev)
 
-## Basic Usage
+## Variables
+Get familiar with the features. 
 
-The TextInput component is used to create form input fields where users can enter text. It supports various states and can be customized using props.
+### Size
+
+**Medium**
+
+This is the default size for text inputs. 
 
 ```jsx-live
-const App = () => <TextInput placeholder="basic text input"/>
+const App = () => <TextInput size="md" value="input value"/>
 ```
 
-### Sizes
+**Small**
 
-Available sizes.
+Available for the times a more condensed text input is needed.
 
 ```jsx-live
-const App = () => {
-
-  const sizes = ['md', 'sm',];
-
-  return (
-    <Stack direction="horizontal" alignItems="center">
-      {sizes.map(size => (
-        <TextInput key={size} size={size} placeholder={size}/>
-      ))}
-    </Stack>
-  )
-}
+const App = () => <TextInput size="sm" value="input value"/>
 ```
 
-### Variants
+### Tone
 
-There are two variants currently:
-- **Solid:** The default variant with a solid background.
-- **Ghost:** A minimal variant without background.
+**Placeholder**
 
+This is the styling for placeholder text. It is a shade lighter than active text.
 
 ```jsx-live
-const App = () => {
-
-  const variants = ['solid','ghost',];
-
-  return (
-    <Stack direction="horizontal">
-      {variants.map(variant => (
-        <TextInput key={variant} variant={variant} placeholder={variant} />
-      ))}
-    </Stack>
-  )
-}
+const App = () => <TextInput size="sm" placeholder="input value"/>
 ```
 
-## States
+**Primary**
 
-### Disabled
+This is the default tone for inputs a user adds or is displayed within a text input.
 
-When interaction is not allowed.
+```jsx-live
+const App = () => <TextInput size="sm" tone="primary" value="input value"/>
+```
+
+**Critical**
+
+To be used when there is an error in form validation.
 
 ```jsx-live
-const App = () => {
-
-  const variants = ['solid','ghost',];
-
-  return (
-    <Stack direction="horizontal">
-      {variants.map(variant => (
-        <TextInput key={variant} variant={variant} placeholder={variant} disabled/>
-      ))}
-    </Stack>
-  )
-}
+const App = () => <TextInput size="sm" tone="critical" isInvalid value="input value"/>
 ```
 
-### Invalid
+## Guidelines
+
+Text input guidelines focus on creating clear, accessible, and user-friendly fields for data entry by emphasizing proper labeling, providing helpful cues and validation, ensuring keyboard navigation, and offering clear feedback on input status.
+
+### Best practices
+
+* **Keyboard Navigation:**
```

Comment:
Are we using asterisks or dashes for bulleted lists? Do we care to be consistent?

Reply:
Honestly, I think dashes have been used more, I'll change it.